gen/perunServicesUtils.pm

```diff
@@ -12,10 +12,10 @@ no if $] >= 5.018, warnings => "experimental"; #supress warnings on experimetal
 
 #use Perun::Agent;
 
-# Get attributes from getData function and convert them into hash
+# Get attributes from getHashed function and convert them into hash
 #
 # Usage:
-# my $data = perunServicesInit::getHierarchicalData;
+# my $data = perunServicesInit::getHashedHierarchicalData;
 # my %facilityAttributes = attributesToHash $data->getAttributes
```

Comment:
I think that this "usage" in docs is not relevant anymore, since `getHashedHierarchicalData()` returns different structure and there is no `getAttributes()` method for it. We might probably think about removal of `attributesToHash()` and `dataToAttributesHashes()` while the first one seems to be used from a single place within the services.

Reply:
As for `dataToAttributesHashes`, we might say it is not needed anymore, so I deleted it. 
However, as you mentioned, `attributesToHash` is used in `pbsmon_json service`, so it depends whether given service is used (needed) or can be deleted as a whole.